### ISSUE TITLE
Several commands in Policy have to be updated

### ISSUE THREAD

**Issue by ypshenychka** [zerocracy/zerocracy.github.io#6]
1. [Here](http://www.zerocracy.com/policy.html#30) instead of ***quality is good*** should be ***quality good***
2. [Here](http://www.zerocracy.com/policy.html#9) instead of ***wait*** should be ***waiting*** (not sure about this one)

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@ypshenychka thanks, fixed in e2059f0

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
com.jcabi.xml.XSLDocument[400] java.lang.IllegalArgumentException: Failed to transform by net.sf.saxon.TransformerFactoryImpl: [Job name "gh:zerocracy/zerocracy.github.io#6" is not valid, Processing terminated by xsl:message at line 27 in file:///; SystemID: file:///; Line#: 27; Column#: 6]
net.sf.saxon.expr.instruct.Message[223] net.sf.saxon.expr.instruct.TerminationException: Processing terminated by xsl:message at line 27 in file:///
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm)
